DailyCodingProblem/Program.cs

```diff
@@ -92,5 +97,61 @@ static void Problem2()
                 Console.Write($"{number} ");
             }
         }
+
+        static void Problem3()
+        {
+            Node root = new Node(
+                0,
+                new Node(
+                    1,
+                    new Node(11, null, null),
+                    new Node(12, null, null)),
+                new Node(
+                    2,
+                    new Node(21, null, null),
+                    new Node(22, null, null)));
+
+            Console.WriteLine(SerializeProblem3(root));
+        }
+
+        static string SerializeProblem3(Node root)
+        {
+            if (root == null)
+            {
+                return "null / ";
+            }
+            string serializedTree = string.Empty;
```

Comment:
It would be more efficient to just go ahead and set the initial value to your nodes value plus the separator